frontend/test/pytest/test_from_plxpr.py

```diff
@@ -216,7 +216,7 @@ def circuit(U):
         qjit_obj(x)
         catalxpr = qjit_obj.jaxpr
         call_jaxpr_pl = converted.eqns[0].params["call_jaxpr"]
-        call_jaxpr_c = catalxpr.eqns[1].params["call_jaxpr"]
+        call_jaxpr_c = catalxpr.eqns[0].params["call_jaxpr"]
```

Comment:
There is a utility function in jax_primitives.py that finds the call_jaxpr without indexing through the equations with magic numbers. I am wondering if it make sense to use the same function here as well.
https://github.com/PennyLaneAI/catalyst/blob/178090082b7e85e57ca7a921f0707a3f8c3694e9/frontend/catalyst/jax_primitives.py#L813

Reply:
I'm resolving this comment as I implemented the changes in #1342